File: 1° Semestre/programacaoDeComputadores/06_04/IMC.py
print("qual seu pedo?")
p= float(input())
print ("Qual sua altura?")
a= float (input())

imc= p/(a*a)
print("seu IMC é de ", imc)
if (imc<= 18.5):
  print("magro")
else:
  if (imc <= 25):
    print("normal")
  else:
    if(imc<=30):
      print("gordinho")
    else:
      print("obeso")

File: 2° Semestre/Lista C/imc.c
#include <stdio.h>

int main(void) 
{
  float peso, altura, imc;
  printf("Qual a sua altura:");
  scanf("%f",&altura);
  printf("Qual a sua peso:");
  scanf("%f",&peso);

  imc=peso/(altura*altura);

  if (imc<=18.5) printf("Magro");
  else if (imc<=25) printf("Normal");
       else if (imc<=30) printf("Sobrepeso");
            else printf("Obeso");
  return 0;
}